content/en/docs/concepts/architecture/controller.md

```diff
@@ -0,0 +1,160 @@
+be completed. The Job controller makes the current state be nearer
+to the desired state, by creating Pods that do the work you
+wanted.
+
+Controllers also update the objects that configure them
+For example: once the work is done for a Job, the Job controller
+updates that Job object to mark it `Finished`.
+
+(This is a bit like how some thermostats turn a light off to
+indicate that the your room is now at the temperature you set).
+
+### Direct control
+
+By contrast with Job, some controllers need to make changes to
+things outside of your cluster.
+
+For example, if you use a control loop to make sure there
+are enough {{< glossary_tooltip text="Nodes" term_id="node" >}}
+in your cluster, then that controller needs something outside the
+current cluster to set up new Nodes when needed.
+
+This kind of controller finds its desired state from the API
+server and then communicates directly with an external system
+to bring the current state closer in line.
+
+(There actually is a controller that horizontally scales the
+nodes in your cluster. See
+[Cluster autoscaling](https://kubernetes.io/docs/tasks/administer-cluster/cluster-management/#cluster-autoscaling)).
+
+## Desired versus current state {#desired-vs-current}
+
+Kubernetes takes a cloud-native view of systems, and is able to handle
+constant change.
+
+Your cluster could be changing at any point as work happens and
+control loops automatically fix failures. This means that,
+potentially, your cluster never reaches a stable state.
+
+So long as the controllers for your cluster are making useful changes,
+it's OK for the current state to be different from the desired state
+at a particular moment.
+
+## Design
+
+As a tenet of its design, Kubernetes uses lots of controllers that each manage
+a particular aspect of cluster state. Most commonly, a particular control loop
+(controller) uses one kind of resource as its desired state, and has a different
+kind of resource that it manages to make that desired state happen.
+
+It's useful to have simple controllers rather than one, monolithic set of control
+loops that are interlinked. Controllers can fail, so Kubernetes is designed to
+allow for that.
+
+For example: a controller for Jobs tracks Job objects (to discover
+new work) and Pod object (to run the Jobs, and then to see when the work is
+finished). In this case something else creates the Job objects, whereas the Job
+controller creates some Pod objects.
+
+{{< note >}}
+There can be several controllers that create or update the same kind of object.
+Behind the scenes, Kubernetes controllers make sure that they only pay attention
+to the resources linked to their controlling resource.
+
+For example, you can have Deployments and Jobs; these both create Pods.
+The Job controller does not delete the Pods that your Deployment created,
+because there is information ({{< glossary_tooltip term_id="label" text="labels" >}})
+the controllers can use to tell those Pods apart.
+{{< /note >}}
```

Comment:
The Job controller does ... that your Deployment created because ...
[labels are used to identify or group specific Pods.]

Reply:
Labels are used for lots of different things, beyond identifying or group specific Pods. I am worried about misleading a reader about the scope of labels.

I hope that the tooltip on `labels` is enough to let an inquiring reader go and learn more about the topic.

Reply:
Also, I want to keep this note as short as is reasonable possible.